src/App.js

```diff
@@ -1,41 +1,72 @@
-import logo from './logo.svg';
 import './App.css';
-import Button from "@material-ui/core/Button";
 import { useEffect, useState } from "react";
+import UsersTable from "./components/usersTable/usersTable";
+import {Box, CircularProgress, FormControl, InputLabel, MenuItem, Select} from "@material-ui/core";
+import ContactsApi from "./api/contactApi";
+
+// const users = [{
+//   name: "John",
+//   email: "example@example.com",
+//   cell: "wtf",
+//   gender: "n/a",
+//   nationality: "earth",
+// }]
 
 function App() {
-  const [counter, setCounter] = useState(0);
-  const handleIncrement = () => {
-    setCounter(counter => counter + 1);
-  }
+  const [filterNationality, setFilterNationality] = useState('');
+  const [filterGender, setFilterGender] = useState('');
+  const [users, setUsers] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const handleSetFilterNationality = (e) => {
```

Comment:
Usually I prefer to have some space between these blocks of code: states / effects / handlers.